src/include/OpenImageIO/fmath.h

```diff
@@ -897,7 +897,7 @@ inline T safe_pow (T x, T y) {
     // less than FLOAT_MAX (but still bigger than anyone is likely to
     // need) so that subsequent additions or multiplications are less
     // likely to overflow and end up with an Inf right afterwards.
-    const T big = T(1e30);
+    const T big = std::numeric_limits<T>::max();
```

Comment:
OK, I'll fix up the comment.

I think that any large constant we pick can have subsequent overflows (e.g. multiply the result of two pow's that got clamped to 1e30, and you still have Inf). Clamping at 1e30 could have cases that get silently clamped to something several orders of magnitude less than their perfectly valid results. It's a tradeoff I don't have a 100% confident answer for. But I think clamping at FLOAT_MAX, even with no wiggle room, is at least an *objective* value if the main goal is just to prevent safe_pow from returning Inf. Any other value is a totally arbitrary choice.